gcc/rust/parse/rust-parse-impl.h

```diff
@@ -12900,6 +12900,12 @@ Parser<ManagedTokenSource>::null_denotation (const_TokenPtr tok,
     case UNSAFE:
       return parse_unsafe_block_expr (std::move (outer_attrs),
 				      tok->get_locus ());
+    case UNDERSCORE:
+      add_error (
+	Error (tok->get_locus (),
+	       "Use of %qs is not allowed on the right-side of an assignment",
```

Comment:
```suggestion
	       "use of %qs is not allowed on the right-side of an assignment",
```

We usually use lowercase error messages, as it looks better with GCC's internal formatter

Reply:
I have made the change. @CohenArthur Thanks for reviewing!